### DISCUSSION TITLE
I have a few requests

### DISCUSSION THREAD

**Discussion by WoodsBeatle** [FossifyOrg/General-Discussion#249] [Ideas]
- ASUS ROG TwinView Dock support
- Customize background individually per contact/group message as well as on dial pad, pretty much on any Fossify app
- Fix a few glitches where notifications won't disappear even after I've seen it; rarely SMS app will freeze
- Volume slider during call screen
- Customizable call screen symbols such as square, triangle, octagon etc...

My phone is a ROG 3

**Comment by Aga-C** [comment]
Please create feature requests or bug reports for these in the proper repositories. It'll make it easier for us to track these items.
